docs/src/doc/decisions-and-ADRs/adr/0009-rtl-support.stories.mdx

```diff
@@ -0,0 +1,32 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Documentation/Decisions and ADRs/ADRs/9) RTL support" />
+
+# 9) RTL support
+
+Date: 2020-07-24
+
+## Status
+
+Accepted
+
+## Context
+The components library has to support Right-to-Left (RTL) languages.
+In the past the challenge of writing styles which work in LTR and RTL languages was solved by tools which take a stylesheet and flip it for a RTL language.
+Such tools include but are not limited to [cssjanus](https://www.npmjs.com/package/cssjanus), [rtlcss](https://www.npmjs.com/package/rtlcss), [rtl-css-js](https://www.npmjs.com/package/rtl-css-js).
+
+In the recent past (2018) [CSS Logical Properties and Values](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties) was introduced. As MDN web docs describe it, it's a module of CSS introducing logical properties and values that provide the ability to control layout through logical, rather than physical, direction and dimension mappings.
+This approach solves the double support for LTR and RTL languages. Instead of writing 2 stylesheets, we can now write one, which defines the correct directionality regardless of the direction of the language.
+
+We identified two downsides:
```

Comment:
```suggestion
We identified two challenges:
```

Not because I like euphemisms so much, but because I think the second point in particular is not something negative, but something we need to get over and which will make us become better devs in the process.